docker-images/kafka/scripts/kafka_connect_config_generator.sh

```diff
@@ -104,6 +104,15 @@ EOF
 )
 fi
 
+if [ "$KAFKA_CONNECT_FILE_CONFIG_PROVIDER" = "true" ]; then
+    FILE_CONFIG_PROVIDER=$(cat <<EOF
+# File config provider
+config.providers=file
+config.providers.file.class=org.apache.kafka.common.config.provider.FileConfigProvider
```

Comment:
@scholzj I thought we already supported this?

Reply:
We support it, but do not enable it by default. User has to enable it in `Kafkaconnect.spec.config` I think that is the right way from security perspective. However I wonder if this could be enabled directly in the Mirror Maker 2 scripts instead of adding this to the Connect scripts. I'm also not sure what exactly is is used for, but I guess it has some reason.

Reply:
I've moved the addition of the file config provider from these scripts to the spec handling code in `KafkaMirrorMaker2Cluster.java`.